plugin/trino-opensearch/src/main/java/io/trino/plugin/opensearch/client/OpenSearchClient.java

```diff
@@ -386,25 +387,26 @@ public List<String> getIndexes()
                 JsonNode root = OBJECT_MAPPER.readTree(body);
                 for (int i = 0; i < root.size(); i++) {
                     String index = root.get(i).get("index").asText();
-                    // make sure the index has mappings we can use to derive the schema
-                    int docsCount = root.get(i).get("docs.count").asInt();
-                    int deletedDocsCount = root.get(i).get("docs.deleted").asInt();
```

Comment:
why have we removed it?

Reply:
We have to check the index access regardless of the doc count hence removed it.

Reply:
I am still not clear, without having access to an index `docs.count` and `docs.deleted` are still returned? asking as I couldn't create such setup.

>// without documents, the index won't have any dynamic mappings, but maybe there are some explicit ones

As comment states, the purpose of `docs.count` and `docs.deleted` is to collect tables not having dynamic mapping defined.

Reply:
I got what you are saying. I will look into it. Thanks for the detailed review.